src/trainer_tower.c

```diff
@@ -79,7 +71,6 @@ EWRAM_DATA struct UnkStruct_203F458 * gUnknown_203F458 = NULL;
 EWRAM_DATA struct UnkStruct_203F45C * gUnknown_203F45C = NULL;
 EWRAM_DATA u8 unused_variable = 0;
 
-void sub_815D96C(void);
 void sub_815DC8C(void);  // setup
 void sub_815DD2C(void);  // teardown
 void sub_815DD44(void);
```

Comment:
A lot should be marked as static in this file, and stuff with external linkage should be declared in the header. And other things should be fixed like indent level for switch case (not essentially in this PR, tho)

Reply:
Will fix in a future PR